src/material/BUILD.bazel

```diff
@@ -1,5 +1,7 @@
 package(default_visibility = ["//visibility:public"])
 
+load("@npm//scss-bundle:index.bzl", "scss_bundle")
```

Comment:
Where is this coming from? I don't see the rule in their source or package. Does rules_nodejs automatically generate this?

Reply:
Yes. It's a new feature 💯 https://github.com/bazelbuild/rules_nodejs/releases/tag/0.38.0